ccpp/data/GFS_typedefs.meta

```diff
@@ -5965,6 +5973,12 @@
   units = flag
   dimensions = ()
   type = logical
+[lightning_threat]
+  standard_name = lightning_threat_indices_enabled
```

Comment:
We're supposed to use "do_X" for flags like this.

Reply:
The variable name is already quite long. I've updated the standard name:

```
[lightning_threat]
  standard_name = do_lightning_threat_index_calculations
  long_name = enables the lightning threat index calculations
  units = flag
  dimensions = ()
  type = logical
```